dependency/dependency.go

```diff
@@ -3,14 +3,14 @@
 // more recent (non-vulnerable) version of the dependency in the go.mod file resolves this issue.
 package dependency
 
-import (
-	// nolint:revive
-	_ "github.com/aws/aws-sdk-go"
-	_ "github.com/containerd/containerd"
-	_ "github.com/emicklei/go-restful"
-	_ "golang.org/x/crypto/argon2"
-	_ "golang.org/x/text"
-	_ "gopkg.in/yaml.v2"
-	_ "k8s.io/client-go"
-	_ "k8s.io/kube-openapi/pkg/validation/spec"
-)
+// import (
```

Comment:
I think these changes are required for proper snyk scan

Reply:
Once we updated the dependencies they were no longer needed.

Reply:
Oh nice! then they can be deleted instead of commenting out